appbasic/src/main/java/io/reark/rxgithubapp/basic/fragments/RepositoriesFragment.java

```diff
@@ -0,0 +1,103 @@
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package io.reark.rxgithubapp.basic.fragments;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import javax.inject.Inject;
+
+import io.reark.rxgithubapp.basic.R;
+import io.reark.rxgithubapp.basic.RxGitHubApp;
+import io.reark.rxgithubapp.basic.activities.ChooseRepositoryActivity;
+import io.reark.rxgithubapp.shared.utils.ApplicationInstrumentation;
+import io.reark.rxgithubapp.shared.view.RepositoriesView;
+import io.reark.rxgithubapp.shared.viewmodels.RepositoriesViewModel;
+
+public class RepositoriesFragment extends Fragment {
+    private RepositoriesView.ViewBinder repositoriesViewBinder;
+
+    @Inject
+    RepositoriesViewModel repositoriesViewModel;
+
+    @Inject
+    ApplicationInstrumentation mInstrumentation;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        RxGitHubApp.getInstance().getGraph().inject(this);
+
+        repositoriesViewModel.getSelectRepository()
+                .subscribe(repository ->
+                        ((ChooseRepositoryActivity) getActivity())
+                                .chooseRepository(repository.getId()));
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.repositories_fragment, container, false);
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        repositoriesViewBinder = new RepositoriesView.ViewBinder(
+                (RepositoriesView) view.findViewById(R.id.repositories_view),
+                repositoriesViewModel);
+        repositoriesViewModel.subscribeToDataStore();
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        repositoriesViewBinder.bind();
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+        repositoriesViewBinder.unbind();
```

Comment:
super.onPause() does not trigger the pausing, afaik onPause is just a callback to be executed entirely before the pausing happens.